subsystems/intakesubsystem.py

```diff
@@ -90,25 +94,17 @@ def periodic(self) -> None:
             # only front - keep intaking
             # front and back - get position and hold
             # only back - go to held position from front and back
-            if frontLimitState and backLimitState:
-                if not self.hasPosition:
-                    self.heldPosition = self.intakeMotor.get(
-                        NEOBrushless.ControlMode.Position
-                    )
-                self.intakeMotor.set(
-                    NEOBrushless.ControlMode.Position, self.heldPosition
-                )
-            elif (
-                not frontLimitState and backLimitState
-            ):  # Note: not only aplies to first part (this is for front off back on)
-                self.intakeMotor.set(
-                    NEOBrushless.ControlMode.Position, self.heldPosition
-                )
-            else:
-                self.intakeMotor.set(
-                    NEOBrushless.ControlMode.Velocity, constants.kIntakeSpeed
-                )
-            self.intakeMotor.set(NEOBrushless.ControlMode.Position, self.heldPosition)
+
+            if self.putInPlace:
+                self.intakeMotor.set(NEOBrushless.ControlMode.Position, self.heldPosition)
```

Comment:
shouldn't this hold position logic exist in other held areas?

Reply:
Yeah, this is far from done but wanted to see if this seems about right for the general idea